src/App/PropertyLinks.cpp

```diff
@@ -1075,7 +1096,22 @@ unsigned int PropertyLinkList::getMemSize() const
     return static_cast<unsigned int>(_lValueList.size() * sizeof(App::DocumentObject *));
 }
 
-DocumentObject *PropertyLinkList::find(const std::string &name, int *pindex) const {
+DocumentObject *PropertyLinkList::find(const char *name, int *pindex) const {
+    if (!name)
+        return nullptr;
+    if(_lValueList.size() <= 10) {
+        for(int i=0;i<(int)_lValueList.size();++i) {

```

Comment:
Range-base loop and a counter? (I could go either way on this one, but old-style loops set off warning sirens for me, and there is prior art in that direction on line 1165).

Reply:
Matched the style in #15501.